src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Invoice/InvoicePaymentDto.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Invoice
+{
+    public class InvoicePaymentDto
+    {
+        public int Amount;
+        public int RemainingCaptureAmount;
+        public int RemainingCancellationAmount;
+        public int RemainingReversalAmount;
+        public InvoicePaymentAuthorizationListDto Authorizations;
+        public ICancellationsListResponse Cancellations;
+        public ICapturesListResponse Captures;
+        public DateTime Created;
+        public DateTime Updated;
+        public string Currency;
+        public string Description;
+        public Uri Id;
+        public string Instrument;

```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)